packages/yew/src/platform/rt_tokio/local_worker.rs

```diff
@@ -0,0 +1,190 @@
+//! We use a local worker implementation that does not produce a JoinHandle for spawn_pinned.
+//! This avoids the cost to acquire a JoinHandle.
+//!
+//! See https://github.com/tokio-rs/tokio/issues/4819
+//!
+//! We will not be able to produce a meaningful JoinHandle until WebAssembly targets support
+//! unwinding.
+
+use std::cell::RefCell;
+use std::future::Future;
+use std::marker::PhantomData;
+use std::sync::Arc;
+use std::{io, thread};
+
+static DEFAULT_WORKER_NAME: &str = "yew-runtime-worker";
+
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+use futures::channel::mpsc::UnboundedSender;
+use futures::stream::StreamExt;
+use tokio::task::{spawn_local, LocalSet};
+
+type SpawnTask = Box<dyn Send + FnOnce()>;
+
+thread_local! {
+    static TASK_COUNT: RefCell<Option<Arc<AtomicUsize>>> = RefCell::new(None);
```

Comment:
Why use `Arc<AtomicUsize>` for a thread local? I think `Cell` (or `Rc<Cell>`) should do the job

Reply:
This needs to be set from the thread that spawns the task so the task count can be increased instantly.